portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsRichTextField.testcase

```diff
@@ -43,13 +43,49 @@ definition {
 	}
 
 	@description = "Verify that a Rich Text Field can be deleted"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanBeDeleted {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
+		
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Rich Text",
+			formPageNumber = "0");
+		
+		FormViewBuilder.closeSidebar();
```

Comment:
@ewertonluna Can you please use `Form.closeSidebar();` to keep consistency since you're using on line 71? Thanks!